tee_supplicant/src/tee_supplicant.c

```diff
@@ -63,7 +63,7 @@ struct param_value {
 
 static int receive_request(const struct device *dev, struct tee_supp_request *ts_req)
 {
-	int rc = tee_suppl_recv(dev, &ts_req->cmd, &ts_req->num_param, ts_req->params);
+	int rc = tee_suppl_recv(dev, ts_req->cmd, ts_req->num_param, ts_req->params);
```

Comment:
please, recheck if we actually need this changes with current Zephyr

Reply:
Prototype of this function:
`typedef int (*tee_suppl_recv_t)(const struct device *dev, uint32_t func, unsigned int num_params, struct tee_param *param);`

This is branch optee_upstr

Reply:
So, I think it can be a single commit in such case.